resources/sass/style.scss

```diff
@@ -159,8 +159,9 @@ small {
   @include fluid-type(320px, 991px, $font-size-min-deco-3, $font-size-deco-3);
 }
 
+.ui-widget-content a,
 a {
-  color: $color-link;
+  color: $color-link!important;
```

Comment:
Do we need "!important" here? Can we make it work without?

Reply:
Bump to change review status.

Reply:
I added the "!important" because when I added it without on Knechtgarten it still took the style from theme.css, another solution would be to add some higher general class to override it with for example .path-node from the body this also works

Reply:
I "fixed" it by removing the original definition from jquery.ui. Due to dependency management by Drupal is was loaded after the style.scss.